.github/workflows/pytest.yml

```diff
@@ -27,8 +27,6 @@ jobs:
         # 6th block: These have some XFAILs
           - |
             --ignore=pymc3/tests/test_distribution_defaults.py
-            --ignore=pymc3/tests/test_distributions.py
```

Comment:
@michaelosthege I probably did something wrong here as I see this on the workflow run:

> The following 2 tests are run multiple times: {'pymc3/tests/test_distributions.py', 'pymc3/tests/test_distributions_random.py'}

The goal was to re-enable the two test files. Should I just have removed the --ignore without adding them below?

Reply:
In addition to adding the lines in the second job definition (you did that), you'll want to move the `--ignore` lines from the 1st to the 3rd block.
Otherwise they run in the 2nd __and__ 1st job definition.

It would probably be clearer if we could put the comments further down. Damn YAML.. Feel free to elaborate the comments!

Reply:
You can run `python scripts/check_all_tests_are_covered.py` manually before pushing.

Reply:
Thanks! Hopefully I got it now